frontend/app/(no-footer)/organiser/gallery/page.tsx

```diff
@@ -0,0 +1,12 @@
+//gallery
+"use client";
+import OrganiserNavbar from "@/components/organiser/OrganiserNavbar";
+
+export default function Gallery() {
+  return (
+    <div className="pt-16 pl-14 max-h-screen">
+      <OrganiserNavbar currPage="Gallery" />
```

Comment:
Is there a way to extract the OrganiserNavbar to the root `Layout.tsx` file? we shouldn't need to add a component on every single page we instantiate

Reply:
i just copied how brian did it lol

Reply:
You can try make a new layout.tsx file in the /organiser subdirectory.

The only challenge you will have to solve is how to inject the curr selected page into the organiserNavbar so that it highlights the correct one.